.circleci/config.yml

```diff
@@ -227,7 +227,11 @@ jobs:
 
   phpcs:
     docker:
-      - image: cimg/php:7.2.34
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v1
```

Comment:
Why are we changing to this installed image?

Reply:
Before the change, we used this job only for PHP CodeSniffer. It doesn't require magento installed
PHPStan requires magento installed (it checked all magento classes)

I decided to use one job for both services because even together they finish before unit tests

Reply:
Just a question then. Doesnt this mean its only checking M2.3? or are we covered for newer versions regardless.

Reply:
I guess its not super important actually. Just wanted to make sure we thought about all of these.

Reply:
We only need Magento to check that we use the correct (existing) Magento classes in the plugin code.
So it's better to check with the oldest Magento version - this way, we can see which classes were created only in new Magento versions.